<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="urunler">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>311</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Mevcut Ürünlerimiz</string>
   </property>
   <widget class="QRadioButton" name="radio_televizyon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>101- Televizyon</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_bilgisayar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>102- Bilgisayar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_saat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>105- Akıllı Saat</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_tablet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>103- Tablet</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_kulaklik">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>106- Kulaklık</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_powerbank">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>107- Powerbank</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_playstation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>108- PlayStation</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_mouse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>109- Mouse</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_telefon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>104- Telefon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="urun_bilgi">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ürün Bilgisi Göster</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Sipariş Oluşturun</string>
   </property>
   <widget class="QLabel" name="sip_id">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sipariş ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="urun_id_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Ürün ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="adet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Adet</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sip_bos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="urun_bos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="adet_bos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fiyat_hesap">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fiyat Hesapla</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sip_olustur">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Siparişi Oluştur</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="stok">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Stok Güncelleme</string>
   </property>
   <widget class="QLabel" name="urun_id2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Ürün ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="urun_id">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="stok_gnclle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stok Adet Güncelle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stok_guncelle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="islem_tam">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>İşlemi Tamamla</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
